renderer/src/components/ActivityLog.tsx

```diff
@@ -65,9 +65,8 @@ const ActivityLog: FC<ActivityLogProps> = ({ activities = [] }) => {
           <div>
             {activities.map((activity) => (
               <ActivityLogItem
-                key={activity.id}
-                id={activity.id}
-                timestamp={activity.timestamp}
+                key={`${activity.date}${activity.type}${activity.source}${activity.message}`}
```

Comment:
`core` currently doesn't expose an `.id`. Should we add that, or is this compound key safe enough? `.id` is _only_ used for this one purpose, in the new architecture.

Reply:
The timestamp has milliseconds, but I'll add the `.id` anyway to be safe 👍

Reply:
https://github.com/filecoin-station/core/pull/96